cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -868,10 +870,25 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                 }
             });
         },
+        getCurrentLogo: function(){
+            return this.$('#lesson_logo').val();
+        },
+        setLogo: function(value) {
+            if (value){
+                this.$('#prev_lesson_logo').attr('src', value); }
+            else {
+                this.$('#prev_lesson_logo').attr('src', "/static/studio/images/hera-default-lesson-logo.png");
```

Comment:
please create a variable for `prev_lesson_logo` element to avoid double search this element in the DOM.

Reply:
done